### DISCUSSION TITLE
v1.3.1

### DISCUSSION THREAD

**Discussion by n4ze3m** [n4ze3m/page-assist#225] [Announcements]
## What's Changed

* Updated Playground UI
* Added code download feature to code blocks
* Added pin/unpin option for chat history
* Grouped chat history by "Today," "Yesterday," "Last 7 days," and "Older"
* Added GGUF model download button to Hugging Face website for the Ollama popup

**Full Changelog**: https://github.com/n4ze3m/page-assist/compare/v1.3.0...v1.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/n4ze3m/page-assist/releases/tag/v1.3.1'>v1.3.1</a>.</em>

**Comment by spj2401Dev** [comment]
Another amazing update. Love to see it. Keep going :D
